exercises/dot-dsl/tests/dot-dsl.rs

```diff
@@ -2,9 +2,9 @@
 extern crate maplit;
 extern crate dot_dsl;
 
+use dot_dsl::graph::Graph;
```

Comment:
I don't think it was necessary to move it as a part of this PR.  I request some explanation be added before merging.

Reply:
You're right, sorry, this wasn't a necessary change. It wasn't intentional: my dev environment is set up to automatically order use statements alphabetically. 

It being unintentional does not mean that it is not potentially useful. I have a preference at strength 2/10 to keep imports ordered in this way, because in larger projects this means that it's easier to scan the list of imports in a predictable way. 

Does this explanation suffice, or would you prefer that I undo this change before merging?

Reply:
Well, if it's not too much trouble, putting it in a separate commit with something similar (easier to scan if it's in a predictable order) will cause me to have no objections. My preference is only 1/10 even if I did object (if no change is made before merging)